### ISSUE TITLE
[a11y] 4.1.2 Name, role, value available

### ISSUE THREAD

**Issue by tschiebel** [owncloud/client#11590]
### Pre-submission Checks

- [X] I [checked for similar issues](https://github.com/owncloud/client/issues?q=), but could not find any. I also checked the closed issues. I could not contribute additional information to any existing issue.
- [X] I will take the time to fill in all the required fields. I know that the bug report may be dismissed otherwise due to lack of information.

### Describe the bug

1

### Expected behavior

1

### Client version number

1
